src/NuGet.Clients/NuGet.CommandLine/Common/Console.cs

```diff
@@ -303,49 +303,50 @@ public void PrintJustified(int startIndex, string text, int maxWidth)
                     }
 
                     // Get the next substring to be printed
-                    text = text.Substring(newLineIndex == -1 ? content.Length : crlf ? content.Length + 2 : content.Length + 1);
+                    text = text.Substring(newLineIndex == -1 ? content.Length : isCRLF ? content.Length + 2 : content.Length + 1);
                 }
             }
         }
 
-
+        // The input string to PrintJustified could use different newline variables than the
+        // client's environment does. The previous implementation used Environment.Newline
+        // to find the index of the newline character, but would fail to do so if there was
+        // a mismatch. This method returns the correct result regardless of the client's environment.
```

Comment:
Looks good, but use the "///" block comment style. In VS, typing "///" will auto generate a summary and parameters (which you can leave blank IMO).

Reply:
Looks like you added a /, but actually want you to use the VS tooling to generate the comment block. See https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Protocol/Resources/MetadataResourceV3.cs#L29

Reply:
I ended up doing away with that method, so I didn't need to comment block in the end, but thanks for bringing this to my attention anyway!